policyengine_us/parameters/gov/states/ky/tax/income/exemptions/homestead/age_threshold.yaml

```diff
@@ -0,0 +1,16 @@
+description: Kentucky qualifies filers for the homestead exemption at or above this age threshold.
+values:
+  2021-01-01: 65
+metadata:
+  period: year
+  unit: year
+  label: Kentucky homestead exemption age threshold
+  reference:
+  - title: Kentucky Department of Revenue - Homestead Exemption
+    href: https://revenue.ky.gov/Property/Residential-Farm-Commercial-Property/Pages/Homestead-Exemption.aspx
+  - title: DOR Sets 2021-2022 Homestead Exemption
```

Comment:
@MaxGhenis this reduces the property taxes, should we still finalize it and include it into a property tax folder